### PR TITLE
Support arm64 for more tests/examples

### PR DESCRIPTION
**Pull request by everpeace**
Currently, for arm64, it only tests sshfs in proxy&starter mode.
This PR extends arm64 support for other examples.

parahaps relates to #5 
 
<details>
<summary>manual e2e test result on darwin/arm64</summary>

```console
❯ uname -ms
Darwin arm64
❯ kind version
kind v0.23.0 go1.21.10 darwin/arm64

❯ make test-e2e
...
./examples/check.sh ./proxy/mountpoint-s3 mfcp-example-proxy-mountpoint-s3 starter /test.txt busybox /data/test.txt
Checking Pod "mfcp-example-proxy-mountpoint-s3"...
pod/mfcp-example-proxy-mountpoint-s3 created
waiting for pod
waiting for pod
waiting for pod
waiting for pod
waiting for pod
waiting for pod
waiting for pod
waiting for pod
waiting for pod
waiting for pod
waiting for pod
waiting for pod
Pod is ready.
FUSE is mounted.
OK.
pod "mfcp-example-proxy-mountpoint-s3" deleted
./examples/check.sh ./proxy/goofys mfcp-example-proxy-goofys starter /test.txt busybox /data/test.txt
Checking Pod "mfcp-example-proxy-goofys"...
pod/mfcp-example-proxy-goofys created
waiting for pod
waiting for pod
waiting for pod
waiting for pod
Pod is ready.
FUSE is mounted.
OK.
pod "mfcp-example-proxy-goofys" deleted
./examples/check.sh ./proxy/s3fs mfcp-example-proxy-s3fs starter /test.txt busybox /data/test.txt
Checking Pod "mfcp-example-proxy-s3fs"...
pod/mfcp-example-proxy-s3fs created
waiting for pod
waiting for pod
waiting for pod
waiting for pod
Pod is ready.
FUSE is mounted.
OK.
pod "mfcp-example-proxy-s3fs" deleted
./examples/check.sh ./proxy/ros3fs mfcp-example-proxy-ros3fs starter /test.txt busybox /data/test.txt
Checking Pod "mfcp-example-proxy-ros3fs"...
pod/mfcp-example-proxy-ros3fs created
waiting for pod
waiting for pod
waiting for pod
waiting for pod
Pod is ready.
waiting for mount
FUSE is mounted.
OK.
pod "mfcp-example-proxy-ros3fs" deleted
./examples/check.sh ./proxy/sshfs mfcp-example-proxy-sshfs starter /root/sshfs-example/test.txt busybox /data/test.txt
Checking Pod "mfcp-example-proxy-sshfs"...
pod/mfcp-example-proxy-sshfs created
waiting for pod
waiting for pod
Pod is ready.
FUSE is mounted.
OK.
pod "mfcp-example-proxy-sshfs" deleted
./examples/check.sh ./starter/ros3fs mfcp-example-starter-ros3fs starter /test.txt busybox /data/test.txt
Checking Pod "mfcp-example-starter-ros3fs"...
pod/mfcp-example-starter-ros3fs created
waiting for pod
waiting for pod
waiting for pod
waiting for pod
waiting for pod
Pod is ready.
FUSE is mounted.
OK.
pod "mfcp-example-starter-ros3fs" deleted
./examples/check.sh ./starter/sshfs mfcp-example-starter-sshfs starter /root/sshfs-example/test.txt busybox /data/test.txt
Checking Pod "mfcp-example-starter-sshfs"...
pod/mfcp-example-starter-sshfs created
waiting for pod
waiting for pod
Pod is ready.
FUSE is mounted.
OK.
pod "mfcp-example-starter-sshfs" deleted
```
</details>

### CODE REVIEW

File: examples/proxy/mountpoint-s3/Dockerfile [hunk 1]
```diff
@@ -25,7 +26,7 @@ set -eux
 EOF
 RUN chmod +x /configure_minio.sh
 
-RUN wget https://s3.amazonaws.com/mountpoint-s3-release/1.0.1/x86_64/mount-s3-1.0.1-x86_64.deb -O mount-s3.deb && dpkg -i mount-s3.deb
+RUN wget https://s3.amazonaws.com/mountpoint-s3-release/latest/${TARGETARCH}/mount-s3.deb -O mount-s3.deb && dpkg -i mount-s3.deb
```
Comment (naoki9911) [inline examples/proxy/mountpoint-s3/Dockerfile:29]: mountpoint-s3 uses 'x86_64' for amd64 environments.

```diff
+ RUN if [ "$TARGETARCH" = "amd64" ]; then TARGETARCH=x86_64; fi && wget https://s3.amazonaws.com/mountpoint-s3-release/latest/${TARGETARCH}/mount-s3.deb -O mount-s3.deb && dpkg -i mount-s3.deb
```

File: Makefile
Comment (naoki9911) [inline Makefile:1]: Could you remove this line?

https://github.com/pfnet-research/meta-fuse-csi-plugin/blob/942093aaf2304c5eae800ef2acec14a8474fdeed/Makefile#L100

**Review by naoki9911 — CHANGES_REQUESTED** [review]

File: Makefile
Comment (everpeace) [reply Makefile:1]: Thanks, fixed in 4dc8859

File: examples/proxy/mountpoint-s3/Dockerfile (hunk 1 above)
Comment (everpeace) [reply examples/proxy/mountpoint-s3/Dockerfile]: Thanks, too. fixed in eb3fed6

**Review by everpeace — COMMENTED** [review]
addressed your comments. PTAL 🙇

**Comment by naoki9911** [comment]
LGTM, Thanks!

**Pull request merged**